resources/web/map/js/trajectory.js

```diff
@@ -102,22 +132,20 @@ new QWebChannel(qt.webChannelTransport, (channel) => {
         }
     });
 
-    chn.recvPos.connect((id, lat, lng) => {
-        const pos = [lat, lng];
-
-        data[id].features.push({
-            type: 'Feature',
-            geometry: {
-                type: 'Point',
-                coordinates: pos
-            }
-        });
-        if (!map) return
+    chn.recvPos.connect((id, lat, lng, hAcc, vAcc) => {
+        const pos = [lat, lng],
+            RADIUS_MIN = 0.0002,
+            // cap radius to be 20cm minimum, for visibility on map
+            rX = Math.max(RADIUS_MIN, hAcc / 1000),
```

Comment:
the map plot should be a circle with radius hAcc, the vAcc is not visible on a top down map view

Reply:
ahhh so it should just be a circle